### PR TITLE
Remove temporal image validation

### PR DESCRIPTION
**Pull request by binoue**
This PR removes temporal image validation

Signed-off-by: binoue <banji-inoue@cybozu.co.jp>

### CODE REVIEW

File: neco-admission/base/deployment.yaml [hunk 1]
```diff
@@ -45,10 +45,6 @@ spec:
         - --valid-image-prefix=gcr.io/datadoghq/agent
         # TODO: remove this after maneki complete the transition to gcr.io
         - --valid-image-prefix=datadog/agent,docker.io/datadog/agent
-        # TODO: remove this after we build our own images for Rook
-        - --valid-image-prefix=k8s.gcr.io/sig-storage/,quay.io/cephcsi/
```
Comment (ymmt2005) [inline neco-admission/base/deployment.yaml:49]: @llamerada-jp @toshipp @satoru-takeuchi 
Could you confirm this?

Comment (binoue) [reply neco-admission/base/deployment.yaml:49]: We are still using `k8s.gcr.io/sig-storage/csi-node-driver-registrar` image , so the validation can not be deleted

**Review by ymmt2005 — CHANGES_REQUESTED** [review]
Please reconfirm the images.

File: neco-admission/base/deployment.yaml (hunk 1 above)
Comment (llamerada-jp) [reply neco-admission/base/deployment.yaml:49]: We are confirmed that and we agree with you.
ref. https://github.com/cybozu-go/neco/issues/940

**Comment by binoue** [comment]
No images containing `datadog/agent`, `docker.io/datadog/agent`, and `uay.io/integreatly/grafana_plugins_init` exists in our staging and production environments

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**